doc/benchmark/modules/ROOT/pages/preparing-keycloak.adoc

```diff
@@ -61,16 +61,24 @@ Some scenarios require a service account with the clientId `gatling`:
 
 Instead of following the above manual steps, you can use this link:{github-files}/benchmark/src/main/content/bin/initialize-benchmark-entities.sh[initialize-benchmark-entities.sh] script to do the setup for you.
 
+Specify the path to Keycloak distribution directory as KEYCLOAK_HOME.
+[source,shell]
+----
+export KEYCLOAK_HOME=provision/keycloak-cli/keycloak
```

Comment:
This `keycloak-cli` directory wont appear until the `task` is run for the first time, which kind of means, the user is forced to use `minikube/provision` module to provision the application. It would be nice to point them to a default installation path of keycloak-cli or where they can download it from the web.

Reply:
The prequisites of this section mentions that there is a running Keycloak installation which in turn implies that keycloak-cli already exists. How about adding the disclaimer that this is only for when you provision using minikube?

Reply:
this would apply for any kind of provisioning of keycloak, such as Stand Alone, docker-compose, Openshift, however the keycloak-cli directory's presence is only guaranteed by the provision/minikube based installation. That's where we wanted to make these instructions to be not dependent on the minikube provisioning